tasks/git-hooks/protected-branches.py

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/python3
+
+import re
+import subprocess
+import sys
+
+
+def main():
+    try:
+        local_branch = subprocess.check_output('git rev-parse --abbrev-ref HEAD', shell=True).decode('utf-8').strip()
+        print(local_branch)
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏Some leftover I guess ?
```suggestion

```

Reply:
You guessed correctly :sweat_smile: removed